app/src/main/java/com/orange/ods/app/ui/modules/Module.kt

```diff
@@ -25,16 +24,25 @@ sealed class Module(
     @DrawableRes val imageRes: Int,
     @StringRes val descriptionRes: Int,
     val route: String,
-    val demoScreen: @Composable (upPress: () -> Unit) -> Unit,
-    val imageAlignment: Alignment = Alignment.Center,
+    val imageAlignment: Alignment = Alignment.TopCenter,
```

Comment:
`Alignment.Center` seems to be working, especially since we use `ContentScale.Crop` to display the module illustrations.